tests/helpers/test_script.py

```diff
@@ -815,7 +815,7 @@ async def test_wait_for_trigger_variables(hass: HomeAssistant) -> None:
     actions = [
         {
             "alias": "variables",
-            "variables": {"seconds": 5},
+            "variables": {"seconds": 0.01},
```

Comment:
`async_wait_for_task_count` was really overly complicated, so it's good to see it go.

I think this change means the test is now stalling for 10ms, which is short enough that the test may be flaky?
The point of the test is to make sure trigger variables work. Can't we achieve that by templating something else in the `wait_for_action` than a friggin timeout?

Reply:
ha yes, I agree. We can just fire an event with a text.